CHANGES.md

```diff
@@ -1,5 +1,6 @@
 ## Version 2.1.1
 
+* Fixed problem with unsupported time format for permafrost datasets #944 
```

Comment:
Afaik, there is no CCI ECV "permafrost". Please say what datasets are concerned, e.g. provide an ID pattern or so, so people can understand what now work and what was broken..